flask_rebar/utils.py

```diff
@@ -0,0 +1,124 @@
+"""
```

Comment:
is utils the right name for this package? should we be more specific? imo, any utils package ends being a dumping ground when we could just be diligent about structuring things.

Reply:
Hrmmm - yeah, I wrestled a bit with that internally; given that it struck you as well (and that in the course of this PR it already evolved a bit past what I initially had in mind) it's probably worth a little refactor before we merge